### ISSUE TITLE
[frio] Add focus styles to buttons

### ISSUE THREAD

**Issue by MrPetovan** [friendica/friendica#8490]
The frio theme is built on Bootstrap, but the custom style for buttons overrides the style on focus. This is an important accessibility feature for disabled and abled people alike and should be reintroduced.

The challenge is that the button color can be customized and as such the focus style should either work with all the possible colors, or include that specific color.

**Comment by AlfredSK** [comment]
Hmm. The 'new post' button has now the color of the top nav bar and not the link color anymore. Is that intended or is it a mishap.

**Comment by MrPetovan** [comment]
It's probably because in my scheme the color of the top nav bar is too close from the link color 😅

More seriously, it is intended. This button color was used in only two places in the whole site design, both with different styles. I removed the exceptions, and now it has the primary button color.

**Comment by AlfredSK** [comment]
I understand. But that's kind of a pity. In my color scheme the link color is clearly different from the color of the nav bar. So the 'new post' button was clearly visible and it looked nicer.

**Comment by MrPetovan** [comment]
Since we're now using the link color to provide a visible outline when buttons are focused, I'm not sure I can restore a link-color button while keeping the outline color consistent across the site.
